custom_components/gree/translations_helper.py

```diff
@@ -19,7 +19,8 @@
 # Only translate the component name for users
 DEFAULT_NAMES = {
     'ru': 'Gree Климат',
-    'en': 'Gree Climate'
+    'en': 'Gree Climate',
+    'pl': 'Gree Climate'
```

Comment:
@Riffaells i'm not sure if this can be moved to the .json's. This feels a bit weird to just put a small part of the translation here. Is it possible to move this to the JSON too?

Reply:
for me it's the same as config.title in json but I've tried to match previous commits so I've added string here too

Reply:
Yeah but i'm more questioning if we should want this small separate language specific configuration outside of the language files.

So: Should we fix this before we add more languages?

Thats what i'm requesting from @Riffaells.